iclr_wrap_up/run_experiments.py

```diff
@@ -0,0 +1,24 @@
+import json
```

Comment:
This could use some abstraction into functions instead of comments and a proper __main__ block.